INSTALL

```diff
@@ -1,236 +1,368 @@
 Installation Instructions
```

Comment:
Please, move INSTALL file changes to a separate commit (or remove them)

Reply:
This is kinda a annoying one, so everytime you configure h-d this file gets updated based on the current auto tools version, so this is sort of a generated file, at the same time we want it in the repo. im not sure what to do about this.

Reply:
anyhow i reverted it for now